code/modules/projectiles/updated_projectiles/ammunition.dm

```diff
@@ -105,7 +105,7 @@ They're all essentially identical when it comes to getting the job done.
 		return
 
 	var/obj/item/ammo_magazine/handful/new_handful = new /obj/item/ammo_magazine/handful()
-	var/MR = caliber == "12g" ? 5 : 8
+	var/MR = caliber == "12g" ? 5 : "7.62x51mm" ? 5 : 8
```

Comment:
```suggestion
	var/MR = caliber == ("12g" || "7.62x51mm") ? 5 : 8
```

Reply:
As you've written there, doesn't work. Pulls out 8 bullets